bang/magicollections.py

```diff
@@ -286,18 +288,25 @@ def share_image(self, context, item):
     def to_fields(self, view, item, exclude_fields=None, *args, **kwargs):
         if exclude_fields is None: exclude_fields = []
         exclude_fields.append('d_names')
+        if item.school is not None:
+            exclude_fields.append('classroom')
+        exclude_fields.append('japanese_name')
```

Comment:
Why are you hiding Japanese name?

Reply:
Oops! I accidentally left that in from a test I was doing earlier. I fixed that ✅